templates/zerver/emails/missed_message.txt

```diff
@@ -9,7 +9,7 @@
 {% else %}
 {% if message_content_disabled_by_realm %}
 {% trans hide_content_url=realm_url + "/help/hide-message-content-in-emails" %}
-This email does not include message content because your organization has disabled message content appearing in email notifications.
+This email does not include message content because your organization hides message content in email notifications.
 See {{ hide_content_url }} for more details.
 {% endtrans %}
 {% elif message_content_disabled_by_user %}
```

Comment:
We should maybe check-in with Alya on CZO about whether the string for the related user setting should also be updated with these changes.